programs/stake/src/stake_state.rs

```diff
@@ -1419,7 +1421,7 @@ fn do_create_account(
 ) -> AccountSharedData {
     let mut stake_account = AccountSharedData::new(lamports, StakeStateV2::size_of(), &id());
 
-    let vote_state = vote_state::from(vote_account).expect("vote_state");
+    let vote_state = VoteState::deserialize(vote_account.data()).expect("vote_state");
```

Comment:
What is this change for?

Reply:
I inlined this utility function to avoid depending on solana-vote-program https://github.com/anza-xyz/agave/blob/1faa6c0d4f2653c877416a66bed7730f2d5e855a/programs/vote/src/vote_state/mod.rs#L151